components/automate-ui/src/app/components/tabs/tabs.component.scss

```
@@ -53,3 +53,33 @@
     background: $chef-light-grey;
   }
 }
+
+.beta-tag {
+  width: 24px;
+  height: 12.8px;
+  left: 937px;
+  top: 221.6px;
+  background: #000000;
```

Should use named colors from _variables.scss:
```suggestion
  background: $chef-black;
```